docs/docs/classic-ml/model-registry/workflow.mdx

```diff
@@ -202,6 +202,102 @@ import os
+
+Additionally, models registered to Databricks Unity Catalog require a signature to be set. If the workspace-registered
+model version does not already have a signature set, we recommend following the [Model Signatures](/ml/model/signatures)
+instructions to add one. However, to ease the burden of migration, you can also set the environment variable
+`MLFLOW_SKIP_SIGNATURE_CHECK_FOR_MIGRATION_TO_DATABRICKS_UC_REGISTRY` to `true` that will work ONLY with the
+
+<APILink fn="mlflow.client.MlflowClient.copy_model_version">`copy_model_version()`</APILink> API as indicated above, and
+allow you to migrate the model to Unity Catalog, bypassing the signature requirement.
+
+```python
+import os
+
+skip_sig_var = "MLFLOW_SKIP_SIGNATURE_CHECK_FOR_MIGRATION_TO_DATABRICKS_UC_REGISTRY"
+os.environ[skip_sig_var] = "true"
+```
+
+NOTE: not having a signature on your Unity Catalog registered model version will limit the available feature set.
+For a full list of what will be unavailable, refer to the [Databricks documentation](https://docs.databricks.com/aws/en/machine-learning/manage-model-lifecycle/#train-and-register-unity-catalog-compatible-models).
+
+If you want to migrate all of the model versions in your workspace registered model to a destination UC model,
+you can use a script as follows:
+
+```python
+import mlflow
+from mlflow import MlflowClient
+from mlflow.models import ModelSignature
+from mlflow.types.schema import Schema, ColSpec, AnyType
+
+workspace_client = MlflowClient(registry_uri="databricks")
+uc_client = MlflowClient(registry_uri="databricks-uc")
+
+
+# Make a dummy model that can be used to increment the version number
+def make_dummy_uri() -> str:
+    class _Dummy(mlflow.pyfunc.PythonModel):
+        def predict(self, ctx, x):
+            return None
+
+    with mlflow.start_run() as run:
+        schema = Schema([ColSpec(AnyType())])
+        model = mlflow.pyfunc.log_model(
+            name="m",
+            python_model=_Dummy(),
+            signature=ModelSignature(inputs=schema, outputs=schema),
+        )
+        return f"models:/{model.model_id}"
+
+
+# Check if the source model has a particular version number
+def src_has_mv(name: str, v: int) -> bool:
+    try:
+        workspace_client.get_model_version(name, str(v))
+        return True
+    except Exception:
+        return False
+
+
+# Copy model versions from a source Databricks workspace-registered model to
+# a destination Databricks Unity Catalog registered model
+def copy_model_versions_to_uc(src: str, dst: str) -> None:
+    latests = workspace_client.get_latest_versions(src)
+    top = max(int(v.version) for v in latests)
+    dummy_uri = make_dummy_uri()
+
+    for v in range(1, top + 1):
+        if src_has_mv(src, v):
+            mlflow.set_registry_uri("databricks")
+            workspace_client.copy_model_version(f"models:/{src}/{str(v)}", dst)
+        else:
+            mlflow.set_registry_uri("databricks-uc")
```

Comment:
We don't need to call `set_registry_uri()` if we're using `MlflowClient` instances

Reply:
Yeah now that we updated to using the store's URI, we don't need to any more